source/geometry_model/spherical_shell.cc

```diff
@@ -210,7 +210,13 @@ namespace aspect
       normal_vector(const typename Triangulation<dim, dim>::face_iterator &face,
                     const Point<dim> &p) const
       {
-        // TODO: Deal with pull back and push forward
+        // We calculate radial, rather than *normal*, vectors here if a face is
+        // at the boundary. This is as described in the documentation of this
+        // function.
+
+        // TODO: If this turns out to ever be wrong, we can always make it
+        //   a parameter whether we want this or the "geometrically correct"
+        //   behavior.
```

Comment:
I think there are cases where one would not want this, see above. So maybe

```suggestion
        // TODO: Add an input parameter that allows it to select
        // whether we want this or the "geometrically correct"
        // behavior.
```